### DISCUSSION TITLE
cfclient running issue

### DISCUSSION THREAD

**Discussion by AlexanderHuang2001** [bitcraze/discussions#679] [General]
I run into some problems when I tried to connect cfclient with crazyflie-drone. Hopefully, you can help me solve this problem.

At first, I installed cfclient in the ubuntu environment. After pressing SCAN button, no radio address is presented. In the console, it says crazyradio is set, but gives a warning "WARNING:cflib.crtp.usbdriver:Exception while scanning for Crazyflie USB: 'NoneType' object has no attribute 'bcdDevice'"

I then followed the instruction and installed cfclient in windows10. Basically same warning comes out, no radio address is detected.

Can you tell me what problem am I facing now?

The error and pictures are attached below. Thanks a lot. Looking forward to your timely reply.

Alex

C:\Users\admin>cfclient
UserWarning: Using SDL2 binaries from pysdl2-dll 2.24.0
INFO:cfclient.gui:Disabling STL printouts
INFO:cfclient.utils.input.inputreaders:Input readers: ['linuxjsdev', 'pysdl2']
INFO:cfclient.utils.input.inputreaders:Could not initialize [linuxjsdev]: Only supported on Linux
INFO:cfclient.utils.input.inputreaders:Successfully initialized [pysdl2]
INFO:cfclient.utils.input.inputinterfaces:Found interfaces: ['leapmotion', 'wiimote', 'zmqpull']
INFO:cfclient.utils.input.inputinterfaces:Could not initialize [leapmotion]: Leap Motion library probably not installed
(No module named 'leapsdk')
INFO:cfclient.utils.input.inputinterfaces:Could not initialize [wiimote]: Missing cwiid (wiimote) driver No module named
 'cwiid'
INFO:cfclient.utils.config:Dist config read from d:\graduation\crazyflie-clients-python\src\cfclient/configs/config.json

INFO:cfclient.utils.config:Config file read from [C:\Users\admin\AppData\Local\Bitcraze\cfclient/config.json]
INFO:cfclient.utils.input.inputinterfaces:Could not initialize [zmqpull]: ZMQ input disabled in config file
INFO:cfclient.utils.zmq_led_driver:ZMQ led disabled in config file
INFO:cfclient.utils.zmq_param:ZMQ param disabled in config file
INFO:cfclient.utils.input:Using device blacklist [(VirtualBox|VMware)]
INFO:cflib.crtp.radiodriver:v0.53 dongle with serial N/A found
INFO:cfclient.ui.tabs.QualisysTab:Switching Flight Mode to: FlightModeStates.DISCONNECTED
INFO:cfclient.utils.input.inputreaders.pysdl2:Looking for devices
INFO:cfclient.utils.input.inputreaders.pysdl2:Found 1 devices
INFO:cfclient.utils.input:Selected MUX: Normal
INFO:cfclient.ui.main:Role of PS4 Controller is Device
INFO:cfclient.utils.input.mux:Adding device PS4 Controller to MUX Normal
INFO:cfclient.ui.tabs.QualisysTab:Exception during qtm discovery: 'QRTDiscoveryProtocol' object has no attribute 'error_
received'
INFO:cflib.drivers.cfusb:Looking for devices....
WARNING:cflib.crtp.usbdriver:Exception while scanning for Crazyflie USB: 'NoneType' object has no attribute 'bcdDevice'
INFO:cflib.crtp.prrtdriver:Initialized PRRT driver.
INFO:cflib.crtp.radiodriver:v0.53 dongle with serial N/A found
INFO:cflib.drivers.cfusb:Looking for devices....
WARNING:cflib.crtp.usbdriver:Exception while scanning for Crazyflie USB: 'NoneType' object has no attribute 'bcdDevice'
INFO:cflib.crtp.prrtdriver:Initialized PRRT driver.
![66042280038566205](https://user-images.githubusercontent.com/68822079/233731826-840387b9-1708-422f-8ef2-2faa7f205d33.jpg)
![屏幕截图 2023-04-22 020003](https://user-images.githubusercontent.com/68822079/233731833-3d883019-aa97-45ed-9fcf-128379725d2d.png)

**Comment by krichardsson** [comment]
Duplicate of #680
